xml/art_solution_automation.xml

```diff
@@ -166,6 +166,163 @@ https://confluence.suse.com/x/8IEnGw
+      </para>
+      <step>
+        <para>
+          Install the &salt; formulas with <command>zypper</command>.
+          Get the package name from the upstream GitHub repository.
+        </para>
+      </step>
+      <step>
+        <para>
+          Create the pillar files structure.
+        </para>
+        <para>
+          The pillar files work as configuration files for the &salt; formulas
+          and are intended as user input. To use the files, create a
+          <filename>top.sls</filename> file which references other files you
+          want to apply.
+        </para>
+        <substeps>
+          <step>
+            <para>
+              Create the directory <filename class="directory">/srv/pillar</filename>
+              and its subdirectories <filename class="directory">hana</filename>,
+              <filename class="directory">netweaver</filename>,
+              and <filename class="directory">drbd</filename>:
+            </para>
+            <screen>&prompt.root;<command>mkdir</command> --parents /srv/pillar/{hana,netweaver,drbd}</screen>
+            <para>
+              The directory names match with the names in the <filename>top.sls</filename>
+              file (see next step).
+            </para>
+          </step>
+          <step>
+            <para>
+              Create the file <filename>/srv/pillar/top.sls</filename> with the following content:
+            </para>
+            <screen language="sls">base:
+    'hana01,hana02':
+        - hana.hana
+        - hana.cluster
+
+    'netweaver01,netweaver02':
+        - netweaver.netweaver
+        - netweaver.cluster
+
+    'drbd01,drbd02':
+        - drbd.drbd
+        - drbd.cluster</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Download the examples from the upstream GitHub repository:
+        </para>
+        <screen>&prompt.user;<command>wget</command> -O hana.sls \
+  https://github.com/SUSE/saphanabootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O netweaver.sls \
+  https://github.com/SUSE/sapnwbootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O drbd.sls \
+  https://github.com/SUSE/drbd-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O cluster.sls \
+  https://github.com/SUSE/habootstrap-formula/raw/master/pillar.example</screen>
+      </step>
+      <step>
+        <para>
+          Copy all pillar files to the right directory:
+        </para>
+        <screen>&prompt.sudo;<command>cp</command> -vi hana.sls cluster.sls /srv/pillar/hana
+&prompt.sudo;<command>cp</command> -vi netweaver.sls cluster.sls /srv/pillar/netweaver
+&prompt.sudo;<command>cp</command> -vi drbd.sls cluster.sls /srv/pillar/drbd</screen>
+      </step>
+      <step>
+        <para>
+          Modify the content of the pillar files depending on your needed configuration
+          of each &salt; formula.
```

Comment:
This reads a little strangely to me.

How about:

         Modify the content of the pillar files for each &salt; formula, according to the configuration you need.

Reply:
Yep, fixed it with commit 5cbc528.